.travis/install.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 set -ex # exit with nonzero exit code if anything fails
 
+pip install --user PyNaCl
```

Comment:
Concerning the python signing tool: 
- Create a file called requirements.txt that contains all required packages. Similar to the documentation requirements. The file should be in the same directory as the python signing tool.
- Replace the line above with
```
pip install  -r <path-to-signing-tool>/requirements.txt
```

Reply:
Used pip3 instead, same as for the doc packages.
However, Travis takes quite some time to install it. Seems like it is building something from source. 
Maybe it has something to do with pip being outdated: "You are using pip version 8.1.1, however version 19.2.1 is available."